packages/sources/eodhistoricaldata/src/adapter.ts

```diff
@@ -1,40 +1,15 @@
-import { Requester, Validator, AdapterError } from '@chainlink/ea-bootstrap'
+import { Builder } from '@chainlink/ea-bootstrap'
 import { ExecuteWithConfig, ExecuteFactory, Config } from '@chainlink/types'
-import { makeConfig, DEFAULT_ENDPOINT } from './config'
+import { makeConfig } from './config'
 import {
   price,
   // , marketCap
 } from './endpoint'
 
-const inputParams = {
-  endpoint: false,
-}
-
 export const execute: ExecuteWithConfig<Config> = async (request, config) => {
-  const validator = new Validator(request, inputParams)
-  if (validator.error) throw validator.error
-
-  Requester.logConfig(config)
-
-  const jobRunID = validator.validated.id
-  const endpoint = validator.validated.data.endpoint || DEFAULT_ENDPOINT
-
-  switch (endpoint.toLowerCase()) {
-    case price.NAME: {
-      return await price.execute(request, config)
-    }
-    // TODO: Re-add
-    // case 'marketcap': {
-    //   return await marketCap.execute(request, config)
-    // }
-    default: {
-      throw new AdapterError({
-        jobRunID,
-        message: `Endpoint ${endpoint} not supported.`,
-        statusCode: 400,
-      })
-    }
-  }
+  const chainlinkEndpoints = [price]
+  // TODO: Readd marketCap endpoint
```

Comment:
What's this?

Reply:
I'm not entirely sure, there was a similar TODO there already (see `develop`) and I just moved it after reformatting the code to use `buildSelector`